### PR TITLE
Bugfix/2269/currency language theme does not exist on mobile

### PR DESCRIPTION
**Pull request by Tolik170**
## Description

1) Fixed language, curreny, theme block styles on mobile at sidebar
2) Fixed color of facebook and instagram icons while changing theme to dark at sidebar

#### Screenshots

|         Original          |         Updated          |
| :-----------------------: | :----------------------: |
| image| image|

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Review by Pipuss — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

File: src/containers/sidebar/sidebar.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ const Sidebar = ({ setIsMenuOpen, isMenuOpen, fromSideBar }) => {
           fromSideBar
           socialIconsStyles={styles.socialIconsStyles}
           position='flex-start'
-          color={isLightTheme ? '#000000' : '#FFFFFF'}
+          color={isLightTheme[0] ? '#000000' : '#FFFFFF'}
```
Comment (tarasSanotskyi) [inline src/containers/sidebar/sidebar.js:157]: use colors from palette

Comment (Tolik170) [reply src/containers/sidebar/sidebar.js]: done.

**Review by MickhaelSh — APPROVED** [review]

**Review by LubomyrKashuba — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

**Review by IrynaMel — APPROVED** [review]

**Review by Pipuss — APPROVED** [review]

**Review by tarasSanotskyi — DISMISSED** [review]

**Review by byavdoshniak — APPROVED** [review]

File: src/containers/sidemenu-right-bar/sidemenu-right-bar.styles.js [hunk 2]
```diff
@@ -25,26 +34,12 @@ export const useStyles = makeStyles((theme) => ({
...
-    '& svg:hover': {
-      color: 'white'
-    }
-  },
-  cartHeader: {
+  wishListAndCartIcon: {
     display: 'flex',
     justifyContent: 'center',
-    transform: 'scale(1.3)',
+    transform: 'scale(1.1)',
```
Comment (LubomyrKashuba) [inline src/containers/sidemenu-right-bar/sidemenu-right-bar.styles.js:40]: use fontSize

**Review by LubomyrKashuba — CHANGES_REQUESTED** [review]

File: src/containers/sidemenu-right-bar/sidemenu-right-bar.styles.js (hunk 2 above)
Comment (Tolik170) [reply src/containers/sidemenu-right-bar/sidemenu-right-bar.styles.js]: Fixed.

**Review by LubomyrKashuba — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by MickhaelSh — APPROVED** [review]

**Review by IrynaMel — APPROVED** [review]

**Review by olga-gchak — APPROVED** [review]

**Pull request merged**